docs/api_call.md

```diff
@@ -1,1352 +1,1154 @@
+      {"data": [
           {
-            "detection": "message", 
-            "original_text": "3", 
-            "entity_value": {
-              "value": "3"
-            }, 
-            "language": "en"
-          }
-        ]
+              "detection": "message",
+              "original_text": "amans.rlx@gmail.com",
+              "entity_value": {"value": "amans.rlx@gmail.com"}
+          }]
       }
+      
       ```
 
-### location
-
-- This functionality calls the TextDetector class to detect location.
-
-- Example:
-
-  - Example 1:
+  - ***Example 2:  Detecting email from fallback value***
 
-    - ```python
-
-      message = 'atm in andheri west'
-      entity_name = 'locality_list'
-      structured_value = None
-      fallback_value = None
-      bot_message = None
-      ```
-
-    - *Python:*
+    - *Django Shell:*
 
       ```python
-      from ner_v1.chatbot.entity_detection import get_location
-      output = get_location(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
+      >>> message = 'send this me to my email'
+      >>> entity_name = 'email'
+      >>> structured_value = None
+      >>> fallback_value = 'amans.rlx@gmail.com'
+      >>> bot_message = None
+      
+      >>> from ner_v1.chatbot.entity_detection import get_email
+      >>> output = get_email(message=message,entity_name=entity_name,
+                             structured_value=structured_value,
+                             fallback_value=fallback_value, bot_message=bot_message)
+      >>> print(output)
+      
       ```
 
     - *CURL command:*
 
       ```shell
       URL='localhost'
       PORT=8081
+      
+      curl -i 'http://'$URL':'$PORT'/v1/email/?message=send%20me%20to%20my%20email&entity_name=email&structured_value=&fallback_value=amans.rlx@gmail.com&bot_message='
+      
       ```
 
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/location/?message=atm+in+andheri+west&entity_name=locality_list&structured_value=&fallback_value=&bot_message='
-      ```
-
-    - *CURL Output:*
+      > **Output **
 
       ```json
-      {
-        "data": [
+      {"data": [
           {
-            "detection": "message", 
-            "original_text": "andheri west", 
-            "entity_value": {
-              "value": "Andheri West"
-            }, 
-            "language": "en"
-          }
-        ]
+              "detection": "fallback_value",
+              "original_text": "amans.rlx@gmail.com",
+              "entity_value": {"value": "amans.rlx@gmail.com"}
+          }]
       }
+      
       ```
 
-### person_name
+  
 
-- This functionality calls the NameDetector class to detect Name entities.
+### 6. Text
 
-- Examples:
+> entity_type: text
 
-  - Example 1:
+The Text Detector has the capability to detect custom text entity within the given text.  The detector has the ability to handle multilanguage text. 
```

Comment:
Incorrect claim `The detector has the ability to handle multilanguage text.` implies the use of multiple languages in the same text. This detector is language agnostic